actions/reconfigure.go

```diff
@@ -346,10 +346,15 @@ func (m *Reconfigure) getBackTemplateProtocol(protocol string, sr *proxy.Service
 	if strings.EqualFold(protocol, "https") {
 		prefix = "https-"
 	}
+	proxy_mode := "http"
```

Comment:
Seems that we added the same feature in parallel. There is the `reqMode` [reconfigure](http://proxy.dockerflow.com/usage/#reconfigure) parameter that serves the same purpose. The major difference is that you implemented it as a global env. var. that will be applied to all services while `reqMode` we have now can differ from one service to another. I'd prefer the current implementation since it provides more flexibility. That does not mean that we cannot use both. There can be an env. var. that specifies the mode that will be applied to all services and could be overwritten with `reqMode`.